### PR TITLE
Add note for cache behavior change in static files

### PR DESCRIPTION
**Pull request by mrkschan**

### CODE REVIEW

**Comment by pjcdawkins** [comment]
This has the old style web config in the YAML example- it should be updated with a new style "locations" block

**Comment by mrkschan** [comment]
Provided example of both format

**Comment by GuGuss** [comment]
I propose we also mention this change in the `expires` key definition with a link to this section.

File: src/configuration/app/upgrading.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+```
+    web:
+        document_root: "/"
+        passthru: "/index.php"
+        index_files:
+            - "index.php"
+        expires: 300
+        whitelist:
+            - \.html$
+```
```
Comment (mrkschan) [inline src/configuration/app/upgrading.md]: updated.

**Pull request merged**